### PR TITLE
Add some logging for cargo pods

### PR DESCRIPTION
**Pull request by optimumtact**
Does what it says.

### CODE REVIEW

**Comment by Timberpoes** [comment]
I'm not particularly well-versed with this code, but would this be a candidate for having the `/datum/supply_pack` handle the logging? Are there any things you can order from the various consoles that don't come from `/datum/supply_pack`?

If all the various cargo-interacting consoles use are supply_packs, it would centralise all logging and prevent an issue like this reoccurring.

Ignore me if I don't know what I'm talking about.

**Comment by optimumtact** [comment]
it's possible, I'll look tonight

**Comment by Timberpoes** [comment]
Genuinely have no clue. I was looking at this code:
image
image

And these comments:
image

And figured this datum was basically the source of all the cargo entries and the spawn source of all filled crates.

It seemed like the most central code to handle logging (I.e. - If this is the code that spawns (and later fills) the physical crate, it's probably the best candidate for logging)

File: code/modules/cargo/expressconsole.dm [hunk 1]
```diff
@@ -220,6 +221,7 @@
 						D.adjust_money(-(SO.pack.get_cost() * (0.72*MAX_EMAG_ROCKETS)))
 
 						SO.generateRequisition(get_turf(src))
+						investigate_log("[key_name(usr)] launched an emagged supply pod at [AREACOORD(LZ)] of type [SO.pack.type]", INVESTIGATE_CARGO)
```
Comment (ShizCalev) [inline code/modules/cargo/expressconsole.dm:224]: compile

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Pull request closed without merging**